mac/Keyman4MacIM/Keyman4MacIM/KMConfiguration/en.lproj/preferences.strings

```diff
@@ -0,0 +1,51 @@
+
+/* Class = "NSButtonCell"; title = "Always show on-screen keyboard"; ObjectID = "4UX-80-0Vo"; */
+"4UX-80-0Vo.title" = "Always show on-screen keyboard";
+
+/* Class = "NSButtonCell"; title = "Use verbose Console logging"; ObjectID = "7J6-zy-R20"; */
+"7J6-zy-R20.title" = "Use verbose Console logging";
+
+/* Class = "NSTabViewItem"; label = "Support"; ObjectID = "93O-x6-RLF"; */
+"93O-x6-RLF.label" = "Support";
+
+/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "C6L-PK-bpq"; */
+"C6L-PK-bpq.title" = "Text Cell";
+
+/* Class = "NSButtonCell"; title = "Download keyboard..."; ObjectID = "CTw-kf-WNS"; */
+"CTw-kf-WNS.title" = "Download keyboard...";
+
+/* Class = "NSWindow"; title = "Keyman Configuration"; ObjectID = "F0z-JX-Cv5"; */
+"F0z-JX-Cv5.title" = "Keyman Configuration";
+
+/* Class = "NSTextFieldCell"; title = "Table View Cell"; ObjectID = "J2x-OZ-TSX"; */
+"J2x-OZ-TSX.title" = "Table View Cell";
+
+/* Class = "NSButtonCell"; title = "< Back"; ObjectID = "JOK-JV-n8w"; */
+"JOK-JV-n8w.title" = "< Back";
```

Comment:
You may want to leave the left arrow `<` out of the string, since right-to-left (RTL) locales will use a right arrow for "back".

If the arrow isn't really needed, maybe leave it out altogether...